data/coalition weapons.txt

```diff
@@ -130,14 +130,14 @@ outfit "Active Finisher"
 		"radar tracking" .8
 		"optical tracking" .5
 		"missile strength" 50
-		"shield damage" 1500
-		"hull damage" 1900
-		"hit force" 240
+		"shield damage" 6000
+		"hull damage" 7600
+		"hit force" 960
```

Comment:
this and the reload change makes Finishers twice as strong damage-wise, I really don't think they need this as they're very strong already
adding that you multiplied the missile strength by 1,5x while it's already very hard to shoot them down (due to their speed too), I think this would make Finishers overpowered